kg_rag/run_setup.py

```diff
@@ -4,10 +4,14 @@
 def download_llama(method):
     from kg_rag.utility import llama_model
     try:
+        if not os.path.exists(config_data["LLM_CACHE_DIR"]):
+            print(f"LLM_CACHE_DIR:  {config_data['LLM_CACHE_DIR']} doesn't exists. Creating one.." )
+            os.makedirs(config_data["LLM_CACHE_DIR"], exist_ok=True)
```

Comment:
Create the `LLM_CACHE_DIR` in the repo. TODO: Update the `.gitignore` file to not check this in the repo.